### ISSUE TITLE
Feature Request: Offer the complete report as JSON

### ISSUE THREAD

**Issue by 1Maxnet1** [mecatran/gtfsvtor#82]
Hello,
for our use case we wanted to have the complete report of as JSON-Output. However currently the JSON-Output only provides a summary, but not the complete report, as stated in the help text: "JSON validation report (summary only for now) output file"

Are there any plans to add the complete report as JSON-Output instead of only a summary?

I would also be interested to contribute to this feature.

**Comment by laurentg** [comment]
No plan to add this feature, but if you want to participate, any contribution welcome!

**Comment by laurentg** [comment]
Some more implementation details. The JSON output is generated by the [JsonReportFormatter](https://github.com/mecatran/gtfsvtor/blob/master/src/main/java/com/mecatran/gtfsvtor/reporting/json/JsonReportFormatter.java.) class. See the convert method, where the method should convert each [ReportIssue](https://github.com/mecatran/gtfsvtor/blob/master/src/main/java/com/mecatran/gtfsvtor/reporting/ReportIssue.java) from the [ReviewReport](https://github.com/mecatran/gtfsvtor/blob/master/src/main/java/com/mecatran/gtfsvtor/reporting/ReviewReport.java).

Please be aware that the JSON export do have "append" mode, so the JSON data report binding class is split in [JsonValidationRun](https://github.com/mecatran/gtfsvtor/blob/b52f19d3569444be0adbd7f78fbf69eb52f334fb/src/main/java/com/mecatran/gtfsvtor/reporting/json/model/JsonReport.java#L42), each containing what should be exported for one validation run (currently only contains categories and a count summary).

In summary, adding full reporting capabilities to the JSON output should be rather straightforward, one just have to complete the JSON model, and loop on each issues to convert them to the JSON java model.
